lib/fathead/c_module_example/parse.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import requests
+import re
+from bs4 import BeautifulSoup
+
+
+BASE_URL    = "http://en.cppreference.com/"
+OUTPUL_FILE = open('output.txt', 'w+')
+MODULE_DICT = {}
+# URLS for each library is provided rather than scraping main pages for links
+RESULT_URL  = ["http://en.cppreference.com/w/cpp/header/cstdlib", "http://en.cppreference.com/w/cpp/header/csignal", "http://en.cppreference.com/w/cpp/header/csetjmp",
```

Comment:
Isn't it better to rather scrape the main pages for links?

Because if they add something new or change these links, we will have to change our links manually.